packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -104,7 +104,7 @@ export async function fetchSnapshotWithRedeem(
             }, error);
             await redeemSharingLink(odspResolvedUrl, storageTokenFetcher, logger);
             const odspResolvedUrlWithoutShareLink: IOdspResolvedUrl =
-                { ...odspResolvedUrl, sharingLinkToRedeem: undefined };
```

Comment:
`            const odspResolvedUrlWithoutShareLink: IOdspResolvedUrl =
                { ...odspResolvedUrl, shareLinkInfo: {...odspResolvedUrlWithoutShareLink.shareLinkInfo,
                    sharingLinkToRedeem: undefined,
                };
            }`
i think it should be like this and then you can remove the below if.

Reply:
@pragya91 to confirm.